src/react_native/core.cljs

```diff
@@ -0,0 +1,29 @@
+(ns react-native.core
```

Comment:
yeah we could rename it later if needed, now naming fit so well, so i would like to keep it like that, because i not sure there will be something else

Reply:
I agree, seems good enough to me as well.

Reply:
I thought we had agreed on libs 🤔  Anyway, we can always nest it later. The idea being that core components/functionality can live there, stuff which is used in both quo2 and status_im 👍